pre-commit.ps1

```diff
@@ -10,7 +10,9 @@ Param (
     $TFTestCliArgs
 )
 
-docker run --rm -v ${pwd}:/lint -w /lint ghcr.io/antonbabenko/pre-commit-terraform:latest run -a
+$tag = "latest"
+docker pull ghcr.io/antonbabenko/pre-commit-terraform:$tag
```

Comment:
pull operation will be done automatically by run operation
also you're duplicating `ghcr.io/antonbabenko/pre-commit-terraform:$tag`
if you want to go this way you can set this as a variable instead of tag that is used just once here